packages/components/table/src/table-body/events-helper.ts

```diff
@@ -103,11 +116,13 @@ function useEvents<T>(props: Partial<TableBodyProps<T>>) {
      *    - Actual: 188.00000762939453
      */
     const rangeWidth = Math.round(range.getBoundingClientRect().width)
-    const padding =
-      (Number.parseInt(getStyle(cellChild, 'paddingLeft'), 10) || 0) +
-      (Number.parseInt(getStyle(cellChild, 'paddingRight'), 10) || 0)
+    const rangeHeight = Math.round(range.getBoundingClientRect().height)
```

Comment:
Calling `Math.round` here seems to have problems in special cases, refer to the comments here https://github.com/element-plus/element-plus/pull/12938#discussion_r1206205475.

Do you have any good advice? @ryuhangyeong